### ISSUE TITLE
Extra options are only ran on initial pbuilder run

### ISSUE THREAD

**Issue by rabejens** [rmohr/chroot-plugin#25]
Any extra command line parameters specified in the chroot configuration are only considered at the first run, i.e., when `debootstrap` runs. The subsequent run, where extra packages are installed, does not take the parameters into account, thus causing the build to fail when a HTTP proxy is specified this way instead of using `.pbuilderrc`.

**Comment by rmohr** [comment]
You are right here. I have always assumed that you have the correct pbuilder infrastructure configuration outside of jenkins.

First I am not sure if we really need something additional. What is your use case? Do you have different proxy setups for different build jobs?

Second I see two ways to solve this:
- An additional arguments field which are always applied
- A textbox where you can copy your .pbuilderrc configuration in
